mne/io/brainvision/brainvision.py

```diff
@@ -498,11 +498,16 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale):
     for chan, props in cfg.items('Channel Infos'):
         n = int(re.findall(r'ch(\d+)', chan)[0]) - 1
         props = props.split(',')
+
         # default to microvolts because that's what the older brainvision
         # standard explicitly assumed; the unit is only allowed to be
         # something else if explicitly stated (cf. EEGLAB export below)
         if len(props) < 4:
-            props += (u'µV',)
+            props += ('µV',)
+        elif props[3] == '':
+            # Deal with files where unit is simply empty.
```

Comment:
OK, this is really minor, but I don't think comments should end with a period, especially if it's just one statement. Also, I think most of us start with lower case.

Reply:
I always try to force myself to kind of create comments that are a little prosaic, trying to construct complete sentences / following common orthographic and grammar rules. Helps me to not just jot down something that's not really helpful, know what I mean? I can change the style though if you insist.

Reply:
"helps me not get too sloppy" are the words I'd been looking for :)